roles/contiv_network/tasks/ovs.yml

```diff
@@ -52,3 +56,11 @@
   with_items:
     - "tcp:127.0.0.1:6640"
     - "ptcp:6640"
+
+- name: disable SE linux
```

Comment:
Please don't disable this for the rest of the project. I did some brief googling but I already have my own tall plate so I can't chase this one down for you. I doubt it's complicated.